src/test/regress/citus_tests/run_test.py

```diff
@@ -154,6 +154,8 @@ def run_python_test(test_name, args):
 
 def run_regress_test(test_name, args):
     original_schedule, schedule_line = find_test_schedule_and_line(test_name, args)
+    print(f"SCHEDULE: {original_schedule}")
+    print(f"SCHEDULE_LINE: {schedule_line}")
```

Comment:
Should we remove those print()s before merging the PR?

Reply:
I think they are still useful output for monitoring purpose.